cli/src/test/scala/com/lightbend/rp/reactivecli/docker/ConfigTest.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2017 Lightbend, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.lightbend.rp.reactivecli.docker
+
+import argonaut._
+import utest._
+
+import Argonaut._
+case class Cfg(
```

Comment:
Can I ask what this class is used for?

Reply:
This mirrors the structure of Docker's config files so that we can simply use Argonaut's `CodecJson.derive` to build a JSON codec that can parse it.